src/routes/webhook/repositories/webhook.entity.ts

```diff
@@ -0,0 +1,174 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  BaseEntity,
+  Generated,
+} from 'typeorm';
+import { TxServiceEvent } from '../../events/event.dto';
+import { SendEventTypes, WebhookPublicDto } from '../dtos/webhook.dto';
+
+@Entity()
+export class Webhook extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ unique: true })
+  @Generated('uuid')
+  publicId: string;
```

Comment:
Instead of creating a new `publicId`, could we replace `id` by a `uuid` generated one? I think it's very confusing to have a `publicId` unique and a autogenerated `id`. We should have one or the other

Reply:
Done https://github.com/safe-global/safe-events-service/pull/398/commits/a82fe25efea48434b71c6d752bfc48b8305bbc3c